mingus/core/chords.py

```diff
@@ -684,7 +726,7 @@ def I7(key):
 
 
 def ii(key):
-    return supertonic(key)
+    return supertonic(key) # note that progressions.to_chords['ii'] will come and pick this up (major 2nd chord due to mingus' classical approach). Instead, that fn has been re-written to try to except lower-casing and add a minor suffix
```

Comment:
Is this comment up-to-date? Seems this is about the changes you didn't keep.

Reply:
Correct, I think we should disregard this now as I removed the notion of jazz chord notation (originally I was using lowercase to imply minor but removed that functionality)